README.md

```diff
@@ -1,65 +1,48 @@
 # gRPC-Go
 
 [![Build Status](https://travis-ci.org/grpc/grpc-go.svg)](https://travis-ci.org/grpc/grpc-go)
-[![GoDoc](https://godoc.org/google.golang.org/grpc?status.svg)](https://godoc.org/google.golang.org/grpc)
+[![GoDoc](https://godoc.org/google.golang.org/grpc?status.svg)][API]
 [![GoReportCard](https://goreportcard.com/badge/grpc/grpc-go)](https://goreportcard.com/report/github.com/grpc/grpc-go)
 
-The Go implementation of [gRPC](https://grpc.io/): A high performance, open
-source, general RPC framework that puts mobile and HTTP/2 first. For more
-information see the [gRPC Quick Start:
-Go](https://grpc.io/docs/languages/go/quickstart/) guide.
+The Go implementation of [gRPC][]: A high performance, open source, general RPC
+framework that puts mobile and HTTP/2 first. For more information see the [Go
+gRPC docs][], or jump directly into the [quick start][].
 
-Installation
-------------
+## Prerequisites
 
-To install this package, you need to install Go and setup your Go workspace on
-your computer. The simplest way to install the library is to run:
+- **[Go][]**, any one of the **three latest major** [releases of Go][].
 
-```
+## Installation
+
+The simplest way to install the grpc-go package is to run the following command:
+
+```console
 $ go get -u google.golang.org/grpc
 ```
 
```

Comment:
Is this now considered the "legacy" method?  If so we should move it after the "module" way.

Reply:
Done